scripts/build-release.ps1

```diff
@@ -35,6 +35,11 @@ function Build {
         Version = $env:nspkg_version_part
     }
 
+    $packages["azure-iot-hub"] = [PSCustomObject]@{
+        File = "setup.py"
```

Comment:
>"setup.py" [](start = 15, length = 10)

This should be `"azure\iot\hub\constant.py"`

Reply:
got it thx